ckanext/canada/tests/test_briefingt.py

```diff
@@ -1,31 +1,33 @@
 # -*- coding: UTF-8 -*-
-from nose.tools import assert_equal, assert_raises
 from ckanapi import LocalCKAN, ValidationError
 
-from ckan.tests.helpers import FunctionalTestBase
-from ckanext.canada.tests.factories import CanadaOrganization as Organization
+import pytest
+from ckanext.canada.tests.fixtures import prepare_dataset_type_with_resources
```

Comment:
do you need this import? I thought pytest had some magic to collect fixtures

Reply:
(like the clean_db one used below)

Reply:
n/a removed custom fixtures